### PR TITLE
CAKE-5372 | Deep copy the units

### PR DESCRIPTION
**Pull request by JoshuaRogan**
## Links

* http://wikia-inc.atlassian.net/browse/CAKE-5372

## Description

* There is a minor mutation to the link in the unit. Copying will resolve any conflicts between loads. 

## Reviewers

@Wikia/cake

### CODE REVIEW

File: app/services/affiliate-slots.js [hunk 1]
```diff
@@ -226,7 +227,7 @@ export default Service.extend({
     currentTargeting.forEach((target) => {
       // we're checking all units
       availableUnits.forEach((unit) => {
-        if (unit.campaign === target.campaign && unit.category === target.category) {
+        if (unit.campaign === 'ddb' && unit.category === 'ddb') {
```
Comment (JoshuaRogan) [inline app/services/affiliate-slots.js]: Oops need to remove

**Review by JoshuaRogan — COMMENTED** [review]
.

**Review by cschucode — APPROVED** [review]
Tested on Dev. LGTM.

File: app/services/affiliate-slots.js (hunk 1 above)
Comment (mklucsarits) [inline app/services/affiliate-slots.js]: Can you explain why DnDBeyond has a special case here?

Comment (JoshuaRogan) [reply app/services/affiliate-slots.js]: It's not, debug code. Thanks for catching

**Pull request merged**